### PR TITLE
implementing waitkey for test instead of sleep

### PR DESCRIPTION
**Pull request by CBaptiste**

### CODE REVIEW

File: Algolia.Search.Test/AlgoliaClientTest.cs [hunk 1]
```diff
@@ -924,5 +924,42 @@ public void TestDnsTimeout()
             var index = _client.ListIndexes();
             Assert.IsTrue(startTime.AddSeconds(0.5) < DateTime.Now);
         }
+
+        private void WaitKey(Index currentIndex, JObject newIndexKey, string updatedKey = null)
```
Comment (redox) [inline Algolia.Search.Test/AlgoliaClientTest.cs]: Maybe you could just `s/updatedKey/updatedACL/` so it's easier to understand?

**Comment by PLNech** [comment]
LGTM 👍

**Comment by CBaptiste** [comment]
done @redox

**Comment by redox** [comment]
👍

**Pull request merged**